tests/integration_tests/performance/test_block_performance.py

```diff
@@ -195,27 +189,35 @@ def read_values(cons, numjobs, env_id, mode, bs, measurement, logs_path):
                     values[measurement_id][value_idx] = []
                 values[measurement_id][value_idx].append(int(data[1].strip()))
 
-    for measurement_id, value_indexes in values.items():
-        for idx in value_indexes:
+    for measurement_id, data in values.items():
+        for time in data:
```

Comment:
Is this a timestamp? If so, would it make sense to preserve it?

Reply:
Since we sample at 1Hz, the index into the samples array is a relative timestamp (e.g. seconds since start of test)